styles.css

```diff
@@ -1,3 +1,3 @@
-h1{
-    color: blue;
-}
\ No newline at end of file
+h1 {
+  color: blue;
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you select the element using a class and not the generic `HTML tag` as specified in the project requirement [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color))